.changeset/gold-eagles-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-yoga/plugin-response-cache': patch
+---
+
+Pass \`context\` to `session` and `buildResponseCacheKey`
```

Comment:
Let's add a code example for using this here and also document this on our docs https://the-guild.dev/graphql/yoga-server/docs/features/response-caching#session-based-caching

Reply:
Done 👍

Reply:
I updated the changeset as well. I didn't add docs about buildResponseCacheKey changes because that one never documented already :/